### PR TITLE
#359 Export to pdf

### PR DESCRIPTION
**Pull request by woksin**
Export to pdf backend functionality should be in place.

Location of datacollector should now appear even if it's an invalid report.

Be aware, testing the export functions in swagger results in (at least for me) corrupted files for some reason.
Do the request by manually writing the route.

I only did local testing of this using localhost:5000/api/casereports/export/pdf and csv

### CODE REVIEW

File: Source/VolunteerReporting/Web/Web.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="DocumentFormat.OpenXml" Version="2.8.1" />
     <PackageReference Include="FakeItEasy" Version="4.1.0" />
+    <PackageReference Include="iTextSharp" Version="5.5.13" />
```
Comment (woksin) [inline Source/VolunteerReporting/Web/Web.csproj]: I think the license for ITextSharp is fine, someone with more insight into that kind of stuff needs to look at that.

**Comment by woksin** [comment]
No one merge yet, there is potentially breaking stuff in here since I added a property to an event, need to hear with @einari

**Comment by Velcrow81** [comment]
Adding properties to events should be safe. Just remember to handle existing read models that dont have the value by adding default vales or make them optional. Correct med when I am wrong @einari 😊. And I am sure dolittle have some fancy way of fixing this through versions etc

**Comment by einari** [comment]
Adding properties are "safe". 
Right now when we have not put it into production - anything goes. 
When we are in production - we need to be a bit more aware. If it's adding a new property and it is not something another BC relies on for meaning till it gets the property itself and is up to speed - it's OK.

We have a generation system that handles generation transitions of events. This needs a bit of attention with regards to inter BC connections.

**Comment by einari** [comment]
What should be a governing thing though; adding properties to events might not necessarily be what you're looking for. Events are not vessels for read models, with added requirements - one should keep an eye on whether or not it is a new event rather than introducing a property as it might be new domain knowledge.

**Pull request merged**